nuitka/ModuleRegistry.py

```diff
@@ -295,6 +295,16 @@ def getModuleOptimizationTimingInfos(module_name):
     return module_timing_infos.get(module_name, ())
 
 
+def sort_key(module):
+    return module.getFullName(), module.kind
+
+
+def insert_module_sorted(module):
```

Comment:
Keeping a structure sorted at insert time is not a bad idea in general, but somehow this looks entirely incomplete, also sory_key is still there, this is never called, there must be stuff missing here.